app/views/credit_cards/show.html.erb

```diff
@@ -11,6 +11,7 @@
     <%= link_to "Reserve card", new_credit_card_reservation_path(@credit_card), class: "btn btn-primary" %>
   <% end %>
   <%= link_to "Back", credit_cards_path, class: "btn btn-primary" %>
+  <%= link_to "Edit", edit_credit_card_path, class: "btn btn-primary" %>
```

Comment:
line 14 needs to be moved above line 10 (else) because only owners can edit a credit card.